Dockerfile

```diff
@@ -93,6 +93,11 @@ RUN /usr/bin/pip3 install widget-periodictable==2.1.2
 RUN /usr/local/bin/jupyter nbextension install --py --user widget_periodictable
 RUN /usr/local/bin/jupyter nbextension enable widget_periodictable --user --py
 
+# Install OPTIMADE.
+WORKDIR /opt/
+RUN git clone https://github.com/aiidalab/aiidalab-optimade.git && cd aiidalab-optimade && git reset --hard e008ca3f00bfbcceea52512e7dfe1c24f803c775
+RUN pip install ./aiidalab-optimade
```

Comment:
Even though you're entering the `aiidalab-optimade` folder in the previous `RUN`, this is disregarded due to the `WORKDIR` command, or?

Reply:
correct, once the command is executed you return back to the folder specified in the previous `WORKDIR` call.